Tests/CreateAPITests/Specs/edgecases.yaml

```
@@ -1001,6 +1007,15 @@ components:
         color:
           type: string
           default: red
+    Image:
+      type: object
+      required:
+        - id
+        - url
+      id:
+        type: string
+      url:
+        type: string
```

Ah, I figured it out... They need to be nested under `properties`

```suggestion
      properties:
        id:
          type: string
        url:
          type: string
```